docs/pages/server-access/architecture.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Teleport Server Access Architecture Overview
+description: Surveying key concepts and architectural concerns when using Teleport Server Access.
+h1: Teleport Server Access
+---
+
+Many administrators will consider several factors when configuring Teleport for their specific stack, architecture, and resources.
+
+This document surveys key concepts and architectural concerns when using Teleport Server Access.
+
+## Server Access security
```

Comment:
This section should probably be moved into the planned Architecture/Security article.

@benarent should we put best practices for TTL in there, into the identity management article, both, something else?

Reply:
Maybe we can created a dedicated section that explains token TTL, and how to deal with secrets in general. We don't do a good job explaining that you can also load tokens from a file and other best practices.

Reply:
Great idea. I haven't gotten to these articles as much as I'd have wanted to yet but I've started to stub that out.